contrib/examples/actions/workflows/orquesta-ask-multiple-parameters.yaml

```diff
@@ -0,0 +1,56 @@
+version: 1.0
+
+description: A basic workflow that demonstrates inquiry with multiple parameters.
+
+tasks:
+  start:
+    action: core.echo message="Automation started."
+    next:
+      - when: <% succeeded() %>
+        do: get_approval
+
+  get_approval:
+    action: core.ask
+    input:
+      schema:
+        type: object
+        properties:
+          approved:
+            type: boolean
+            description: "Continue?"
+            required: True
+          department_id:
+            type: number
+            description: "Your department ID:"
+            required: True
+          extra_output:
+            type: string
+            description: "Your message to echo next if you approve to continue:"
+            default: "None"
```

Comment:
This is minor. But you can just use `None` as in NoneType instead of literal `"None"` which is a string.

Reply:
I will change that as I actually prefer it. I just didn't know from the docs if that's supported, thanks.

Reply:
This failed but I'm not sure if this is intended or a bug. 

`extra_output` does not appear in the response object (e.g. `"response":{"approved":true,"department_id":41411}`) even if a default is defined. So any check on `task(get_approval).result.response.extra_output` fails with `"YaqlEvaluationException: Unable to resolve key 'extra_output' in expression ...`